packages/vscode-extension/src/project/metro.ts

```diff
@@ -228,12 +233,11 @@ export class Metro implements Disposable {
     return websocketAddress;
   }
 
-  private async fetchDebuggerURL() {
-    // query list from http://localhost:${metroPort}/json/list
-    const list = await fetch(`http://localhost:${this._port}/json/list`);
-    const listJson = await list.json();
-    // with metro, pages are identified as "deviceId-pageId", we search for the most
-    // recent device id and want want to use special -1 page identifier (reloadable page)
+  private lookupWsAddressForOldDebugger(listJson: any) {
```

Comment:
Nit: improve types for documentation. Not sure if last two keys are always present and what's the type of `reactNative`.
```
type DebuggerPage = {
  id: string; // format: deviceId-pageId – pre 0.76 (number, -1 | 1), post 0.76 (hex string, number)
  webSocketDebuggerUrl: string;
  title: string;
  reactNative: ???;
}
```

We could also collapse two functions into one and return `[isNewDebugger, Url]` tuple to flatten those ifs below (at little alloc cost).